src/ngraph/runtime/backend_manager.cpp

```diff
@@ -155,6 +155,7 @@ DL_HANDLE runtime::BackendManager::open_shared_library(string type)
 
     string library_name = "lib" + to_lower(type) + "_backend" + string(SHARED_LIB_EXT);
     string my_directory = file_util::get_directory(find_my_file());
+    my_directory += "/../backend/";
```

Comment:
This looks way scary. Perhaps we should have an API where a framework can tell us where to look for backends.

Reply:
I'm open to all suggestions.  Just wanted to get something working.  This probably isn't ready for primetime